src/Viewer.module.css

```diff
@@ -51,8 +51,7 @@
 }
 
 .canvasPanel {
-  flex-grow: 0;
-  max-width: var(--content-width);
+  width: var(--content-width);
```

Comment:
Prevents compression from other components fighting with the canvas panel for space.